palo_alto_pan_os/komand_palo_alto_pan_os/actions/check_if_address_object_in_group/action.py

```diff
@@ -0,0 +1,72 @@
+import komand
+from .schema import CheckIfAddressObjectInGroupInput, CheckIfAddressObjectInGroupOutput, Input, Output, Component
+# Custom imports below
+from komand.exceptions import PluginException
+from komand_palo_alto_pan_os.util.ip_check import IpCheck
+
+class CheckIfAddressObjectInGroup(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='check_if_address_object_in_group',
+                description=Component.DESCRIPTION,
+                input=CheckIfAddressObjectInGroupInput(),
+                output=CheckIfAddressObjectInGroupOutput())
+
+    def run(self, params={}):
+        group_name = params.get(Input.GROUP_NAME)
+        address_to_check = params.get(Input.ADDRESS_TO_CHECK)
+        device_name = params.get(Input.DEVICE_NAME)
+        virtual_system = params.get(Input.VIRTUAL_SYSTEM)
+
+        xpath = f"/config/devices/entry[@name='{device_name}']/vsys/entry[@name='{virtual_system}']/address-group/entry[@name='{group_name}']"
+        response = self.connection.request.get_(xpath)
+
+        # Get the contents of the address group to check and extract all the address object names
+        # Make the call and get the address group
+        ip_objects = response.get("response", {}).get("result", {}).get("entry", {}).get("static")
+        if not ip_objects:
+            raise PluginException(cause="PAN OS returned an unexpected response.",
+                                  assistance=f"Could not find group {group_name}, or group was empty. Check the name, virtual system name, and device name.\ndevice name: {device_name}\nvirtual system: {virtual_system}",
+                                  data=response)
+
+        # Extract all the address objects from the address group
+        self.logger.info(f"Number of IP objects received: {len(ip_objects)}")
```

Comment:
I think this reads better
```suggestion
        self.logger.info(f"Searching through {len(ip_objects)} address objects.")
```